src/app/(sidebar)/smart-contracts/contract-explorer/components/ContractStorage.tsx

```diff
@@ -60,18 +65,81 @@ export const ContractStorage = ({
     );
   }
 
+  const parsedKeyValueData = () => {
+    return storageData.map((i) => ({
+      ...i,
+      keyJson: i.key ? decodeScVal(i.key) : undefined,
```

Comment:
will this be a standard data type? should we create a type for this? and use that type on line `85`?